Localize/bump-versions.js

```diff
@@ -0,0 +1,114 @@
+const fs = require('fs');
+const path = require('path');
+const run = require('../ci/ci-util').run;
+const semver = require('semver');
+
+const currentSprint = parseInt(process.env['SPRINT']);
+
+function getChangedFilesList() {
+    return run('git --no-pager diff --name-only origin/master..Localization').split('\n');
+}
+
+function getTaskOrPackagePaths(paths, rootFolder, exclusions) {
+    let pathSet = new Set();
+    paths.filter(p => p.startsWith(rootFolder))
+        .map(p => p.slice(0, p.indexOf('/', rootFolder.length)))
+        .filter(p => fs.existsSync(path.resolve(p)) && fs.statSync(path.resolve(p)).isDirectory())
+        .forEach(p => pathSet.add(p));
+
+    for (let excludedTask of exclusions) {
+        pathSet.delete(`${rootFolder}${excludedTask}`);
+    }
+    return Array.from(pathSet);
+}
+
+function getTasksPaths(paths) {
+    const taskFolder = 'Tasks/';
+    const exclusions = ['Common'];
+
+    return getTaskOrPackagePaths(paths, taskFolder, exclusions);
+}
+
+function getCommonPacks(paths) {
+    const packFolder = 'common-npm-packages/';
+    const exclusions = ['_download', 'node_modules', 'build-scripts'];
+
+    return getTaskOrPackagePaths(paths, packFolder, exclusions);
+}
+
+function bumpTaskVersion(taskPath, minor) {
```

Comment:
I believe we don't need this anymore

Reply:
We can do this, but please consider this
https://github.com/microsoft/azure-pipelines-tasks-common-packages/pull/51#discussion_r1046346042

Reply:
ok, sounds good